lib/tasks/organisations.rake

```diff
@@ -10,4 +10,17 @@ namespace :organisations do
       OrganisationsFetcher.new.call
     end
   end
+
+  desc "Add organisation that is not in GOV.UK organisation database"
+  task :create, %i[name] => :environment do |_, args|
+    usage_message = 'usage: rails organisations:create["<name>"]'.freeze
```

Comment:
I might be misunderstanding this usage message - running `rails organisations:create["Department of Seasonal Events"]` returns `zsh: no matches found: [Department of Seasonal Events]` for me

Reply:
Ah yes, it works in bash for me too. And it works in zsh when you escape the square brackets:

```zsh
bundle exec rails organisations:create\["Department of Seasonal Events"\]
```

Reply:
That's good to know. I hope it might also work with `"bundle exec rails organisations:create[Department of Seasonal Events]"`.

Thanks for testing it!

Reply:
Just tested `bundle exec rails "organisations:create[Department of Seasonal Events]"` and can confirm it works 👍